tests/spyre_util.py

```diff
@@ -333,7 +333,7 @@ def create_random_request(
             mm_placeholders=None,
             sampling_params=sampling_params,
             eos_token_id=None,
-            arrival_time=0,
+            arrival_time=time.time(),
```

Comment:
I would suggest using `time.monotonic()` instead to avoid issues with daylight savings etc.